dd-sdk-android/src/main/kotlin/com/datadog/android/log/Logger.kt

```diff
@@ -217,14 +218,25 @@ internal constructor(internal var handler: LogHandler) {
 
         /**
          * Enables your logs to be sent to the Datadog servers.
-         * You can use this feature to disable Datadog logs based on a configuration or an application flavor.
+         * You can use this feature to disable Datadog logs based on a configuration or an
+         * application flavor.
          * @param enabled true by default
          */
         fun setDatadogLogsEnabled(enabled: Boolean): Builder {
             datadogLogsEnabled = enabled
             return this
         }
 
+        /**
+         * Sets a minimum priority for the log to be sent to the Datadog servers. If log priority
+         * is below this one, then it won't be sent. Default value is -1 (allow all).
+         * @param minLogPriority Minimum log priority to be sent to the Datadog servers.
+         */
+        fun setDatadogLogsMinPriority(minLogPriority: Int): Builder {
```

Comment:
the way I see it this is something similar to `setVerbosity` feature for our `devLogger` right ? I think it will be really useful for our users.

Reply:
yes, indeed. Users may want to see verbose (all) information in the logcat, but send only certain levels to Datadog.